code/src/WijmoProvider/Features/DirtyMark.ts

```diff
@@ -3,8 +3,7 @@ namespace WijmoProvider.Feature {
     export class DirtyMark
         implements
             OSFramework.Feature.IDirtyMark,
-            OSFramework.Interface.IBuilder
-    {
```

Comment:
@JoaoFerreira-FrontEnd The automation in Github are failing. Please check the feedback